app/components/Onboarding/Steps/ConnectionDetails.js

```diff
@@ -1,10 +1,21 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { FormattedMessage } from 'react-intl'
-import { Box } from 'rebass'
-import { Bar, Form, Header, Input } from 'components/UI'
+import { Box, Flex } from 'rebass'
+import { Bar, Form, Header, Input, Button } from 'components/UI'
+import WithSelectDir from 'components/withSelectDir'
 import messages from './messages'
 
+const SelectFileButton = ({ onClick }) => (
+  <Button size="medium" ml={10} onClick={onClick}>
```

Comment:
The default button type of a button is `submit`. We should probably set the `type` prop as `button`.